pinot-query-runtime/src/test/resources/queries/Aggregates.json

```diff
@@ -644,13 +644,13 @@
         "description": "nested aggregation",
         "sql": "SELECT min(max(int_col)) FROM {tbl}",
         "comments": ".*Aggregate expressions cannot be nested.",
-        "expectedException": "Error composing query plan for.*"
+        "expectedException": ".*Aggregate expressions cannot be nested"
```

Comment:
The error message now is better than before. In fact, we shouldn't have been asserting the original message.